functions-and-scope.js

```diff
@@ -40,20 +59,47 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // * Hoe zorgt ik ervoor dat ik alle waardes uit de array kan langslopen, ook als de array wel 100 entries zou bevatten?
 // Log het antwoord in de terminal.
 
+const hoeveelheid = grades.length;
+let totaal = 0;
+for (let i = 0; i < grades.length; i++){
+    totaal = totaal + grades[i];
```

Comment:
Dit kun je ook wat korter schrijven
```suggestion
    totaal += grades[i];
```